src/Randomizer.Data/Configuration/ConfigFiles/ResponseConfig.cs

```diff
@@ -555,6 +555,11 @@ public class ResponseConfig : IMergeable<ResponseConfig>, IConfigFile<ResponseCo
         public SchrodingersString LocationCleared { get; init; }
             = new SchrodingersString("Cleared {0}.");
 
+        /// <summary>
+        /// Gets the phrases for
```

Comment:
Forgot this?

Reply:
Whoops. Well this certainly didn't explain everything.